pywbem/cim_obj.py

```diff
@@ -5211,7 +5213,25 @@ def tomof(self, indent=0, maxline=MAX_MOF_LINE):
 
 class CIMParameter(_CIMComparisonMixin):
     """
-    A CIM parameter in a method (declaration).
+    A CIM parameter (value or declaration).
+
+    This object can be used as parameter value in the
+    :meth:`~pywbem.WBEMConnection.InvokeMethod` operation, and as a parameter
+    declaration in a :class:`~pywbem.CIMMethod` object.
+
+    For parameter values in method invocations:
+
+      * The `value` attribute is the actual value of the parameter.
+      * Qualifiers are not allowed.
+
+    For parameter declarations in method declarations:
+
+      * The `value` attribute is ignored.
+      * Qualifiers are allowed.
+
+    Scalar (=non-array) parameters and items in array parameters may have a
+    value of NULL (= `None`), any primitive CIM data type, reference type, and
```

Comment:
Should last word in sentence be or rather than any?

Reply:
clarify comment.  Should last work on line be or rather than and

Reply:
Done.